examples/multiple.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Alma Widgets multiple configuration example</title>
+
+    <link rel="stylesheet" href="../dist/widgets.umd.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.min.css"
+    /> -->
+  </head>
+  <body>
+    <div class="body">
+      <div>
+        <img src="../src/assets/alma.svg" class="alma-logo" />
+      </div>
+      <div class="menu">
+        <a href="./basic.html">Basic page</a>
+      </div>
+      <div class="main">
+        <div class="image">
+          <img src="./sac-a-main.jpeg" />
+        </div>
+        <div>
+          <div class="product">
+            <h1>Sac à main</h1>
+            <select onchange="generateRenderer()" id="quantity">
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+              <option value="10">10</option>
+            </select>
+            <h3>450 €</h3>
+            <p>Deluxe edition, bigger price, same product</p>
```

Comment:
not sure to understand this ?

Reply:
Just for fun and to have a little separation between the title and widget. I can remove this if you want